src/main/resources/stubs/strings/strings.gobra

```diff
@@ -177,13 +178,15 @@ pure func Join(elems []string, sep string) (res string) /*{
 // HasPrefix tests whether the string s begins with prefix.
 pure
 ensures ret == (len(s) >= len(prefix) && s[0:len(prefix)] == prefix)
+decreases _
```

Comment:
```suggestion
decreases
```

Reply:
I think it would be good to be consistent, i.e., to use `decreases _` for abstract functions and `decreases` (or an appropriate non-constant measure) for non-abstract functions

Reply:
I thought about using `decreases` and dismissed it because these functions are imported and re-verified in every client. It feels wasteful to reverify termination everywhere. Note that, while the same could be said about reverifying the body, including the definition of the function may be useful for clients (and, unfortunately, Gobra/Viper does not allow us to define a function without verifying its correctness).

Reply:
I'm not sure whether we already do that but we could in principle turn decreases measures for imported pure functions into wildcard measures to avoid the additional overhead of re-verifying termination. I'd rather do this within Gobra as opposed to these stubs such that we can still verify the stubs (and thus check that they actually terminate)